samples/contrib/kubeflow-e2e-mnist/kubeflow-e2e-mnist.ipynb

```diff
@@ -9,15 +9,15 @@
     "In this notebook you will create e2e mnist Kubeflow Pipeline to perfom:\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.        # api_version = 'serving.kubeflow.org/v1beta1"'
</pre><p>Whoops, a typo there with the stray quotation mark.. Can you remove the <code>"</code> after v1beta1 </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/kubeflow/pipelines/pull/7355/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>